### PR TITLE
Add systemd service file

### PR DESCRIPTION
**Pull request by slokhorst**
Fedora and Debian (and most other Linux distributions) have moved to systemd, so it makes sense to provide a systemd service for them.

I have removed the Fedora initscript, since all supported Fedora versions use systemd. Debian 8 has systemd, but Debian 7 is still supported, so I have left the Debian initscript.

I have tested the systemd service file on Arch Linux and Fedora.

### CODE REVIEW

**Comment by jogu** [comment]
Your login all makes sense to me!

Unfortunately there's been a conflict with one of the other pull requests - would you mind rebasing your branch onto the latest code please?

File: motion.service.in [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Motion daemon
+After=local-fs.target
```
Comment (jogu) [inline motion.service.in]: I know absolutely nothing about systemd etc - but I wonder if we should be waiting until the network is available as well as the fs? Not sure how motion will react if it's started before network interfaces are brought up.

**Comment by slokhorst** [comment]
You're right about the network, added that!
I also rebased against master.

**Comment by Mr-Dave** [comment]
I do not suppose there is a quick description that you can post into a issue here for how to setup / configure the system so we can put it into the motion guide?

**Comment by slokhorst** [comment]
The user should put `motion.service` in `/usr/lib/systemd/system/` (maybe we can even install it there by default when you run `make install`?).

Motion can then be started with `systemctl start motion.service` and enabled (start at boot) with `systemctl enable motion.service`.

**Pull request merged**